imap_processing/idex/tests/test_l1_cdfs.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+import xarray as xr
+from cdflib.xarray import cdf_to_xarray
+from cdflib.xarray.xarray_to_cdf import ISTPError
+
+from imap_processing.idex.idex_packet_parser import PacketParser
+
+
+@pytest.fixture()
+def decom_test_data():
+    return PacketParser("imap_processing/idex/tests/imap_idex_l0_20230725_v01-00.pkts")
+
+
+def test_idex_cdf_file(decom_test_data):
+    # Verify that a CDF file can be created with no errors thrown by xarray_to_cdf
+    with tempfile.TemporaryDirectory() as tmpdir:
+        file_name = decom_test_data.write_l1_cdf(
+            version="01", description="", directory=tmpdir
+        )
+        assert file_name == os.path.join(tmpdir, "imap_idx_l1_20250724_0000_p_v01.cdf")
```

Comment:
This is a bit cryptic how you know that this is the proper filename. Should we add some of these identifiers to the `write_l1_cdf` routine so that routine can take in dates, software versions? Or another possibility is to make the string here be composed from those attributes `imap_idx_{level}_{date}_{version?}_{identifer?}_{version?}`
```suggestion
        assert file_name == str(tmp_path / "imap_idx_l1_20250724_0000_p_v01.cdf")
        assert Path(file_name).exists()
```
We should also assert that the file exists?